frontend/tests/e2e/helpers/schoolUtils.js

```diff
@@ -0,0 +1,213 @@
+const {getToken} = require('./generateToken');
```

Comment:
Can we confirm that all functions are used in this file? 

Did a quick look that 
getFacilityTypeDisplayValue 
getSchoolCategoryDisplayValue 
getSchoolGradeListFormatted 
getSchoolOrganizationDisplayValue 
getSchoolMailingAddress

are not used. Removing unused functions would help reduce the amount of code to review.

Reply:
These functions have been tested and will come in handy when we start writing school tests. But let me know if you still want them pulled.

Reply:
I'm thinking we can't use them because of the route issue. Plus we are likely going to hardcode the expected values that for the frontend. So we don't need to fetch this information.  

I'm leaning towards removing them to prevent the confusion.

Reply:
Unused functions removed